cmd/gentypes/gentypes.go

```diff
@@ -479,9 +491,24 @@ var typesExcludeList = map[string]bool{
 }
 
 func main() {
+	flag.Parse()
+
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
-	inputFilename := os.Args[1]
+	if flag.NArg() != 1 {
+		fmt.Fprintln(os.Stderr, "Path to the DAP specification json file is required.")
+		fmt.Fprintln(os.Stderr, "go run cmd/gentypes/gentypes.go <path/to/debugProtocol.json>")
```

Comment:
Nit: this printout will look odd if the user builds/installs a binary of this tool. Maybe use `os.Args[0]` or just something generic like `<gentypes> <path/...>`?

Reply:
done